common/rpc/interceptor/telemetry.go

```diff
@@ -271,6 +274,26 @@ func (ti *TelemetryInterceptor) emitActionMetric(
 			metrics.ActionCounter.With(metricsHandler).Record(1, metrics.ActionType("command_BatchMarkers"))
 		}
 
+		var updates int64
+		for _, msg := range completedRequest.Messages {
+			if msg == nil || msg.Body == nil {
+				continue
+			}
+			body, err := msg.Body.UnmarshalNew()
+			if err != nil {
+				continue
+			}
+			switch body.(type) {
+			case *updatepb.Acceptance:
+				updates += 1
+			case *updatepb.Rejection:
+				updates += 1
+			}
+		}
+		if updates > 0 {
+			metrics.ActionCounter.With(metricsHandler).Record(updates, metrics.ActionType("command_UpdateWorkflowExecution"))
```

Comment:
Is this some const used by billing? Can it be `message_UpdateWorkflowExecution`?

Reply:
I don't believe it's a const used by billing. I saw that their's are like `command:<comand type>`. I can change it to message 🤷 I wasn't sure if being more accurate here is beneficial for the user ("why are all of them 'commands' except for this one?").